src/session/cookies.ts

```diff
@@ -42,8 +42,8 @@ export default class Cookies {
         this._getAllCookiesSync    = this._syncWrap('getAllCookies');
         this._putCookieSync        = this._syncWrap('putCookie');
         this._removeCookieSync     = this._syncWrap('removeCookie');
-        this._removeCookiesSync    = this._syncWrap('removeCookies');
         this._removeAllCookiesSync = this._syncWrap('removeAllCookies');
+        this.syncCookies           = [];
```

Comment:
I think we need to rename this property to `pendingSyncCookies` and make it private.
Also, we need takePendingSyncCookies method, something like this:
```js
takePendingSyncCookies () {
    const cookies = this.pendingSyncCookies;
    
    this.pendingSyncCookies = [];

    return cookies;
}
```